src/megameklab/com/util/Mech/DropTargetCriticalList.java

```diff
@@ -123,7 +123,9 @@ public void mousePressed(MouseEvent e) {
 
                 final Mounted mount = getMounted();
                 if ((e.getModifiersEx() & InputEvent.ALT_DOWN_MASK) != 0) {
-                    changeWeaponFacing(!mount.isRearMounted());
+                    if (canRearMount(mount)) {
```

Comment:
Does this need the similar `!(getUnit() instanceof BattleArmor)` here as below?

Reply:
It won't hurt anything. I suspect there are issues with other unit types, but I'm trying to stick close to the specific mech issue.